packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -478,6 +478,28 @@ describe('Table', () => {
             expect(element.checkValidity()).toBeTrue();
         });
 
+        it('allows row hover styling when not scrolling', async () => {
+            await connect();
+            await element.setData(simpleTableData);
+            await waitForUpdatesAsync();
```

Comment:
nit: is this `waitForUpdatesAsync();` needed? I'd expect that we could do all the setup and then only wait for it to take effect before asserting. Same for the first one (and maybe two) in the test below.